platform/darwin/src/MGLThreadPool_Private.h

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+
+#include <mbgl/util/default_thread_pool.hpp>
+
+@interface MGLThreadPool : NSObject
+@end
+
+@interface MGLThreadPool (Private)
```

Comment:
Thanks; as per the comment above, I'm planning on removing `MGLThreadPool` altogether, since it's a singleton anyway.